packages/kn-plugin-workflow/pkg/command/template/SonataFlow-Builder.containerfile

```diff
@@ -39,7 +39,7 @@ RUN /home/kogito/launch/build-app.sh ./resources
 #=============================
 # Runtime Run
 #=============================
-FROM registry.access.redhat.com/ubi9/openjdk-17-runtime:latest
+FROM registry.access.redhat.com/ubi9/openjdk-17-runtime:1.23
```

Comment:
@ricardozanini @domhanak @saumya1singh @mareknovotny

I have configured the image stream 1.23 in this file too.

Why?
OSL 1.37.0 and sources in current branch are "certified" / "tested" under that image stream.
And thus, when users adds the --with-persistence flag to create the project, the automatically generated Docker file for the gitops profile is prepared to use that image stream.

Customer installtions that wants to use a higher stream, when availble, can change that generated file manually to any future availabe stream at their own risk.

@ricardozanini are we on the same page?